xbmc/interfaces/builtins/GUIBuiltins.cpp

```diff
@@ -375,6 +378,18 @@ static int ToggleDirty(const std::vector<std::string>&)
   return 0;
 }
 
+/*! \brief Toggle display HDR On/Off (equivalent to Windows HDR switch in display settings).
+ *  \param params (ignored)
+ */
+static int ToggleDisplayHDR(const std::vector<std::string>&)
+{
+#if defined(TARGET_WINDOWS)
```

Comment:
please avoid using ifdefs in common sources

Reply:
cannot be easily avoided

Reply:
but possible. nobody said that it's easy. we did a lot of work to clean up the code from ifdefs, please don't produce them again

Reply:
Done